### ISSUE TITLE
no more `leg.polyline` since v5.6.0

### ISSUE THREAD

**Issue by noerw** [public-transport/hafas-client#186]
With `hafas-client@5.5.0` everything worked great.
After updating to 5.6.2, polyline is always `undefined` when calling
```
client.journeys(origin, destination, { polylines: true })
```

edit: only tried with DB profile

**Comment by derhuerst** [comment]
Thanks for reporting!

With the DB profile, `hafas-client@5.5.1` sends a `request.ver` of `1.15`, and `hafas-client@5.6.0` sends a `request.ver` of `1.16`. HAFAS returns the polyline in a different way:

```js
// request.ver: 1.15
{
	"type": "JNY",
	"dep": {
		"locX": 0,
		// …
	},
	"arr": {
		"locX": 2,
		// …
	},
	"jny": {
		"jid": "1|244757|0|80|27072020",
		"prodX": 0,
		"dirTxt": "Kiel Hbf",
		// …
		"polyG": {
			"polyXL": [0],
			"layerX": 0,
			"crdSysX": 0
		},
		"ctxRecon": "T$A=1@O=Berlin Hbf (tief)@L=8098160@a=128@$A=1@O=Hamburg Hbf@L=8002549@a=128@$202007271038$202007271234$ICE  696$$1$$$",
		// …
	},
	// …
}
```

```js
// request.ver: 1.16
{
	"type": "JNY",
	"icoX": 0,
	"dep": {
		"locX": 0,
		// …
	},
	"arr": {
		"locX": 2,
		// …
	},
	"jny": {
		"jid": "1|244757|0|80|27072020",
		"prodX": 0,
		"dirTxt": "Kiel Hbf",
		// …
		"poly": {
			"delta": true,
			"dim": 3,
			"type": "WGS84",
			// …
		},
		"ctxRecon": "T$A=1@O=Berlin Hbf (tief)@L=8098160@a=128@$A=1@O=Hamburg Hbf@L=8002549@a=128@$202007271038$202007271234$ICE  696$$1$$$",
		// …
	},
	// …
}
```

`hafas-client` currently doesn't parse the 2nd variant.

**Comment by derhuerst** [comment]
Will published to npm this soon.

**Comment by derhuerst** [comment]
Published as `hafas-client@5.6.3`.
